account_invoice_import/wizard/account_invoice_import.py

```diff
@@ -481,51 +481,41 @@ def invoice_already_exists(self, commercial_partner, parsed_inv):
             ], limit=1)
         return existing_inv
 
-    def _search_import_config(self, partner, company_id):
-        return self.env['account.invoice.import.config'].search([
-            ('partner_id', '=', partner.id),
-            ('company_id', '=', company_id)])
+    def get_parsed_invoice(self):
+        """Hook to change the method of retrieval for the invoice data"""
+        return self.parse_invoice(self.invoice_file, self.invoice_filename)
 
-    def _get_missing_config(self, partner, company):
-        configs = partner.invoice_import_ids.filtered(
-            lambda r: r.company_id == company)
-        if not configs:
-            raise UserError(_(
-                "Missing Invoice Import Configuration on partner '%s'."
-            ) % partner.display_name)
-        return configs[0].convert_to_import_config()
-
-    def _get_child_company(self, company_id, parsed_inv):
```

Comment:
Why are you removing this?

Reply:
Looks suspicious indeed.

Reply:
Yeah, I see. Sorry for that, will rebase correctly.

Reply:
Is it good now?